selfhost/typechecker.jakt

```diff
@@ -5257,6 +5257,70 @@ struct Typechecker {
         return None
     }
 
+    function typecheck_format(mut this, args: [(String, CheckedExpression)]) throws {
+        guard args[0].1 is QuotedString(val: format_string, span) else{
+            eprintln("Internal Compiler Error: `typecheck_format` called with dynamic format string: `{}`", args[0].1)
+            abort()
+        }
+        
+        mut n_args = 1uz
+        mut index = 0uz
+        mut in_format_sepcifier = false
```

Comment:
```suggestion
        mut in_format_specifier = false
```

Reply:
I cant write..

Good catch, will fix it up later this evening